.github/workflows/release-upload-to-wporg.yml

```
@@ -23,7 +23,7 @@ env:
 jobs:
   get-and-validate-release-asset:
     name: Get intended release details
-    runs-on: ubuntu-latest
+    runs-on: ${{ fromJSON('{"group":"WooCommerce Release Checks"}') }}
```

actionlint (1.7.12)

[error] type of expression at "runs-on" must be string or array but found type "{group: string}"

(expression)